controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -81,9 +83,9 @@ protected void shutDown() throws Exception {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "shutDown");
         try {
             log.info("Stopping REST server listening on port: {}", this.restServerConfig.getPort());
-            channel.close();
+            final GrizzlyFuture<HttpServer> shutdown = httpServer.shutdown(30, TimeUnit.SECONDS);
```

Comment:
Giving some time here to complete existing calls so that shutdown is graceful. shutdownNow will terminate ongoing requests immediately.